content/posts/team_intro.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Introducing the Team"
+description: "Introducing CNeuroMod Team Members"
+date: 2020-05-31 
+author: "Valentina Borghesani"
+image: "team_intro/team.png"
+draft: false
+tags: ["team"]
+---
+
+Hi! I'm Valentina, the latest recruit of CNeuroMod and thus clearly the right person to introduce you to this talented herd of nerds. 
+
+<a href="url"><img src="team_intro/ScienceDay2020Pic.png" align="right" height="30%" width="30%" ></a>
```

Comment:
this link is broken. The path `team_intro/` should be ommitted. Also the html formatting looks weird, especially on the blog preview. I suggest to use a simple markdown insert instead:
```
![ScienceDay2020](ScienceDay2020Pic.png)
```

Reply:
Also, regarding the images themselves, the color is slightly different from the rest of the site. 
It would look better if you integrated the exact shade of green: #36ad5d